server/app.js

```diff
@@ -110,7 +112,10 @@ app.use(methodOverride())
 let mongodb
 const mongoURI = process.env.MONGODB_URI
 const mongodbPromise = co(function* () {
-  return yield MongoClient.connect(mongoURI, config.database.mongo.server)
+  return yield MongoClient.connect(mongoURI, {
+    ssl: false,
```

Comment:
Same here, shouldn't this only be in development?

Reply:
👍 
[c2c8c19](https://github.com/AMP-SCZ/dpdash/pull/367/commits/c2c8c19e9583c3411f0036557c60bc1b65bc357a)